### PR TITLE
Kaggle

### PR DESCRIPTION
**Pull request by garlicbulbxian**

### CODE REVIEW

File: scripts/luna/LUNA_train.py [hunk 1]
```diff
@@ -161,11 +162,19 @@ def get_parser():
         parser.error('Required to set --config_file')
 
     if not args.train_path:
-        parser.error('Required to set --working_path')
+        parser.error('Required to set --train_path')
```
Comment (tomowind) [inline scripts/luna/LUNA_train.py]: This is a small thing, but you don't need this if-statement. You can give "required=True" in creating argparse. See https://docs.python.org/3/library/argparse.html#required

File: scripts/ec2/ec2_launcher.py [hunk 2]
```diff
@@ -57,7 +57,6 @@ def get_parser():
 
     if working_dir is None:
         logger.info('No pending task to be processed. Exit the program.')
-        s.close()
```
Comment (tomowind) [inline scripts/ec2/ec2_launcher.py:60]: I don't know much about sqlalchemy, but it's a good practice to use with-statement. For example,

with s.begin_nexted():
    s.do_something()
    s.commit()

I can't find any good example for sql alchemy though. I only found this link/: http://docs.sqlalchemy.org/en/latest/orm/session_transaction.html

File: scripts/kaggle/nodule_detection.py [hunk 3]
```diff
@@ -44,7 +44,7 @@ def predict(model_id, model_weights, input_path, output_path):
     print('-' * 30)
     print('Predicting masks on test data...')
     print('-' * 30)
-    f_list = glob(input_path + "*.npy")
+    f_list = glob(input_path + os.pathsep + "*.npy")
```
Comment (tomowind) [inline scripts/kaggle/nodule_detection.py]: Use os.path.join(input_path, "*.npy") for these things. Then, you don't have to care about trailing spaces.

**Comment by tomowind** [comment]
My comments are not big things, so don't care too much about them. I will try to read all the merged codes as much as possible and will try to find some bugs hopefully. :)

**Comment by oakcreek** [comment]
Good point. Thanks! That's what caused Chen's error last night.

File: scripts/ec2/ec2_launcher.py (hunk 2 above)
Comment (garlicbulbxian) [reply scripts/ec2/ec2_launcher.py:60]: Good point. I will take a look and change this.

File: scripts/kaggle/nodule_detection.py (hunk 3 above)
Comment (garlicbulbxian) [reply scripts/kaggle/nodule_detection.py]: I thought it only works to construct concrete paths. Let me give a try and let you know.

**Pull request merged**